src/features/CourtCaseDetails/Tabs/Panels/HearingDetailsTable.tsx

```diff
@@ -0,0 +1,52 @@
+import { Hearing } from "@moj-bichard7-developers/bichard7-next-core/build/src/types/AnnotatedHearingOutcome"
+import { format } from "date-fns"
+import { Table } from "govuk-react"
+
+interface HearingDetailsTableProps {
+  hearing: Hearing
+}
+
+interface HearingDetailsTableRowProps {
+  header: string
+  value: string | null | undefined
+}
+
+const HearingDetailsTableRow = ({ header, value }: HearingDetailsTableRowProps) => (
+  <Table.Row>
+    <Table.Cell>
+      <b>{header}</b>
+    </Table.Cell>
+    <Table.Cell>{value}</Table.Cell>
+  </Table.Row>
+)
+
+export const HearingDetailsTable = ({ hearing }: HearingDetailsTableProps) => {
+  const getFormatedDateOfHearing = (dateOfHearing: Date | string) =>
+    dateOfHearing instanceof Date
+      ? format(dateOfHearing, "dd/MM/yyyy").toString()
+      : format(new Date(dateOfHearing), "dd/MM/yyyy").toString() ?? ""
+
+  const getLanguage = (language: string) => (language === "D" ? "D (Don't know)" : language)
```

Comment:
It would be good to investigate if there are any other options we need to add here?

Reply:
Nice one! I'll do it for language & if it applies to anything else, we'll be able to extend.

Reply:
Done for languages & court codes :-)